app/views/spree/admin/orders/index.html.haml

```diff
@@ -1,137 +1,33 @@
 - content_for :page_title do
   = t('.listing_orders')
 
+- content_for :minimal_js, true
+
 - content_for :page_actions do
   %li
     = button_link_to t('.new_order'), spree.new_admin_order_url, icon: 'icon-plus', id: 'admin_new_order'
 
 = render partial: 'spree/admin/shared/order_sub_menu'
 
-- content_for :main_ng_app_name do
-  = "ofn.admin"
-
-- content_for :main_ng_ctrl_name do
-  = "ordersCtrl"
-
 - content_for :table_filter_title do
   = t(:search)
 
 - content_for :table_filter do
   = render partial: 'filters'
 
-.row.index-controls{'ng-show' => '!RequestMonitor.loading && orders.length > 0'}
-  %div{style: "display: flex; justify-content: space-between;"}
-    .ofn-drop-down-with-prepend
-      .ofn-drop-down-prepend{"ng-class": "selected_orders.length == 0 ? 'disabled' : ''"}
```

Comment:
I took it out at the beginning because it was angular-based and it was broken without angular but I didn't didn't get round to adding it back as it needed quite a bit of thought to reimplement it because of the way angular works.

Angular is very **active** and tends to hold lots of objects in memory and constantly watch and re-evaluate and re-render things even if the page isn't changing. It can get really slow if there's a lot of objects. Stimulus is kind of the opposite of that, it's mostly **passive** and event-based and relies a bit more on what's in the DOM instead of holding lots of references in memory. It's very different!

One of the things that was making me hesitate was that I really wanted to not add anything into the Stimulus controller which handles the checkboxes and "check all" functionality that would make it less generic (and less re-usable) by coupling it to specific content or markup on the orders index page. It took a bit of thinking. I've implemented the `2 orders selected` type feedback in a separate controller, used the Stimulus `valueChanged` callbacks system in the checkboxes controller and handled the communication between them with custom DOM Events. What do you think?

Also I noticed the checkboxes controller was using the `every()` function which it turns out is from `lodash` library. I swapped it out for a pure javascript alternative, as I think we'll want to drop `lodash` at some point.